File: LB-Assignments/Assignment_4/program5.c
#include<stdio.h>

int FactDiff(int iNo)
{
    int iCnt = 0;
    int iSumFact = 0;
    int iSumNonFact = 0;
    
    for(iCnt = 1; iCnt < iNo; iCnt++)
    {
        if(iNo % iCnt != 0)
        {
            iSumNonFact += iCnt;
        }
    }
    for(iCnt = 1; iCnt < iNo; iCnt++)
    {
        if(iNo % iCnt == 0)
        {
            iSumFact += iCnt;
        }
    }
    return (iSumFact - iSumNonFact);
}

int main()
{
    int iValue =0;
    int iRet =0;

    printf("enter number : ");
    scanf("%d",&iValue);

    iRet =FactDiff(iValue);

    printf("%d",iRet);

    return 0;
}

File: LB-Assignments/Assignment_22/program5.java
/*
Input : 8
Output : 2  4  6  8  10  12  14  16 
*/
import java.util.*;

class Logic
{
    int iNo = 0;

    public Logic(int a)
    {
        this.iNo = a;
    }

    void Pattern()
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print((2*iCnt)+"\t");
        }
        System.out.println();
    }
}
class program5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue =0;

        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Logic lobj = new Logic(iValue);

        lobj.Pattern();

    }
}